compute/k8s-services/vars.tf

```diff
@@ -901,6 +901,12 @@ variable "helm_exporter_repo_branch" {
   default     = null
 }
 
+variable "helm_exporter_target_namespaces" {
+  type        = string
+  description = "target namespaces filter"
+  default     = ".*"
```

Comment:
I am not sure, but does this support wildcards? I think a blank string represents all namespaces.

Reply:
The string is given a value when it is referenced in the EKS pipeline in the .hcl file though or am i on the wrong lines?

Reply:
Yes, but is this the right default value when it is not set in the HCL file.

Reply:
Oh I see what you are saying so would you say it doesn't require a default value because it is referenced there?

Reply:
Ok gotcha